<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SystemMaintence</class>
 <widget class="QWidget" name="SystemMaintence">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(27, 150, 208);</string>
  </property>
  <widget class="QPushButton" name="programupdade_pb">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>120</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>3</string>
   </property>
   <property name="text">
    <string>3.程序升级</string>
   </property>
  </widget>
  <widget class="QPushButton" name="screencalibration_pb">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>20</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>1</string>
   </property>
   <property name="text">
    <string>1.触摸校准</string>
   </property>
  </widget>
  <widget class="QPushButton" name="languageselection_pb">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>70</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>2</string>
   </property>
   <property name="text">
    <string>2.时间设置</string>
   </property>
  </widget>
  <widget class="QPushButton" name="passwordsetting_pb">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>170</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>4</string>
   </property>
   <property name="text">
    <string>4.密码设置</string>
   </property>
  </widget>
  <widget class="QPushButton" name="paymentinstallment_pb">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>220</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>5</string>
   </property>
   <property name="text">
    <string>5.分期付款</string>
   </property>
  </widget>
  <widget class="QPushButton" name="systemlog_pb">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>270</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>6</string>
   </property>
   <property name="text">
    <string>6.系统日志</string>
   </property>
  </widget>
  <widget class="QPushButton" name="restorefactory_pb">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>320</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>7</string>
   </property>
   <property name="text">
    <string>7.恢复出厂</string>
   </property>
  </widget>
  <widget class="QPushButton" name="currentprogramversion_pb">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>370</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>8</string>
   </property>
   <property name="text">
    <string>8.程序版本</string>
   </property>
  </widget>
  <widget class="QPushButton" name="back_pb">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>420</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>9</string>
   </property>
   <property name="text">
    <string>9.返回菜单</string>
   </property>
  </widget>
  <widget class="QLabel" name="visionMark_lb">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>375</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="sysInfo_wd" native="true">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>220</y>
     <width>151</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget{
background-color: rgb(201, 202, 202);
}
QPushButton{
border-radius:10;
background-color: rgb(80, 80, 80);
font-size:18px;
color:yellow;
}
QPushButton::pressed{
background-color: rgb(120, 120, 120);
}</string>
   </property>
   <widget class="QPushButton" name="sysSetting_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>系统配置(F0)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sysUnlock_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>系统解锁(F1)</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
